rc_dev

```diff
@@ -12,3 +12,4 @@ export PUBLIC_URL=//public.dev.bgdi.ch
 export PRINT_URL=//service-print.dev.bgdi.ch
 export PROXY_URL=//service-proxy.dev.bgdi.ch
 export WMTS_URL=//tod.dev.bgdi.ch
+export ADMIN_URL_REGEXP="^(ftp|http|https):\/\/(.*(\.bgdi|\.geo\.admin)\.ch)|(https?:\/\/localhost:[0-9]{1,9})"
```

Comment:
Is the maximal number of port contains 9 numbers?

Reply:
The port number is an unsigned 16-bit integer which means that the maximum port number is 2^16-1=65535. I changed it in the last commit.